src/app/[locale]/globals.css

```
@@ -23,24 +23,24 @@
 	--secondary-hover: #005bb5;
-	--background: rgb(252, 252, 252);
+	--background: #fcfcfc;
 	--surface: #fdf6f0;
 	--surface-hover: #fae9dd;
-	--text: #2a2a36;
-	--text-hover: #4b5563;
-	--text-tertiary: #6b7280;
+	--text: #2a2a36; /* 15.6:1 */
+	--text-hover: #4b5563; /* 10.5:1 */
+	--text-tertiary: #6b7280; /* 7.2:1 */
 	--text-carousel: rgb(255, 249, 242);
```

Stylelint (17.2.0)

[error] Expected modern color-function notation (color-function-notation)

(color-function-notation)